test/crystalphase.jl

```diff
@@ -1,5 +1,5 @@
 # include("../src/CrystalShift.jl")
-
+# TODO: if this should be in the testing suite, add tests and remove plots and @time macros
```

Comment:
If this file tests important functionality, let's think about including some `@test` macros, so that we can have this in the testing suite we execute before integrating new changes.